salt/client/__init__.py

```diff
@@ -2002,6 +2094,7 @@ def pub_async(
                 # If not, we won't get a response, so error out
                 if listen and not self.event.connect_pub(timeout=timeout):
                     raise SaltReqTimeoutError()
+                log.warn("SENDING %r", payload_kwargs)
```

Comment:
Yep, I'll be removing this logging code and the one above (or providing some better logging :thinking: )

Reply:
gone